manifests/stage-nv-peer-mem-driver/0010_nv-peer-mem-driver-ds.yaml

```diff
@@ -78,8 +78,11 @@ spec:
             - name: NO_PROXY
               value: {{ .RuntimeSpec.NoProxy }}
           volumeMounts:
-            - name: run-mlnx-ofed
-              mountPath: /run/mellanox/drivers
+            - name: ofed-usr-src
```

Comment:
you alway mount the OFED on host?

Reply:
Could you please clarify what you mean? We always mount driver sources to `/run/mellanox/drivers/` within nvPeerDriver POD. If we want to use the driver from the host OS we mount:
HOST `/usr/src` -> nv-peer-mem POD `/run/mellanox/drivers/usr/src`
HOST `/lib/modules` -> nv-peer-mem POD `/run/mellanox/drivers/lib/modules`
If we want to use driver deployed by `Mellanox OFED` POD (it deployes driver to /run/mellanox/drivers path on host) we mount:
HOST `/run/mellanox/drivers/usr/src` -> nv-peer-mem POD `/run/mellanox/drivers/usr/src`
HOST `/run/mellanox/drivers/lib/modules` -> nv-peer-mem POD `/run/mellanox/drivers/lib/modules`

Reply:
sorry this change it not relevant for the fix it accurate mount point right? can you do it in different PR or commit ?

Reply:
I moved this change to separate commit